ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/configs/DynamoDBConfigs.java

```diff
@@ -96,4 +96,11 @@
 	@DefaultValue("xxx:8111")
 	String getDaxEndpoint();
 	
+	
+	@PropertyName(name = "AWSAccessKey")
```

Comment:
Using application.properties means that the credentials have to go into a file curated by this project. Could we use DefaultCredentialProviderChain, so that if run on managed compute, credentials get pulled from the instance profile, and otherwise get pulled from environment variables?

Reply:
Right, we are also picking up the credentials from the instance profile. This seems to be breaking our internal deployment as well. I will into how to use `DefaultCredentialProviderChain`.

Reply:
lets make an issue to fix credentials later

Reply:
I have been using the plugin for some time now and this breaks our internal bindings with AWSCredentials. To move forward with the rest of your work, I will merge it and will revisit on Monday.